src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.module.css

```diff
@@ -0,0 +1,113 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: 465px;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: block;
+  }
+}
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+  }
+}
+
+.textCardWrapper {
+  text-align: center;
+}
+
+@media (--laptop-and-up) {
+  .bleedImage {
+    position: absolute;
+  }
+
+  .gridAreaCard {
+    grid-column: 8 / span 5;
+    z-index: 1;
+  }
+
+  .textCardWrapper {
+    margin-top: 87px;
+  }
+
+  .playIcon {
+    height: 92px;
+    width: 92px;
+  }
+}
+
+.playButton {
+  left: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.textCard {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  padding: 70px 60px;
+}
+
+.eyebrow {
+  color: var(--color-gray-600);
+  font: var(--font-action);
+  text-transform: uppercase;
+}
+
+.description {
+  font: var(--font-body-large);
+  margin-top: 15px;
+}
+
+.actionRow {
+  margin-top: 30px;
+}
+
+@media (--tablet-and-down) {
+  .bleedImage {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+  }
+
+  .textCard {
+    padding: 30px;
+  }
+
+  .title {
+    margin-top: 15px;
+  }
+
+  .image {
+    width: 100%;
+  }
+
+  .playButton {
+    top: initial;
+    transform: initial;
+  }
```

Comment:
I think this, combined with the flex positioning from the `.bleedImage`, is making the positioning of this element difficult to reason about. The way it currently renders, it looks like the play button is off-center, slightly to the right.

I think it'd also be a good idea to stick to absolute positioning and transforming the element to achieve centering, rather than mixing both flex and absolute position when attempting to center the play button.

Also, could you organize your CSS so thing that are related to each other are closer to each other? It's a bit difficult to read your CSS right now because you have components within the `bleedImage` and the `bleedMainContent` mixed together, and you have the media queries for `--tablet-and-down` for the `.playButton` very far away from the `--laptop-and-up` and the non-media-query version.

Could I suggest that you group your CSS selectors to more closely match the hierarchical grouping of elements in the HTML? The two main subsections of the component are the `.bleedImage` and the `.bleedMainContent`. I think it would be helpful if you put all the classes related to the `.bleedImage` together and all the classes related to the `bleedMainContent` together.

Reply:
Pair programmed with developer and resolved